src/test/groovy/test/LoginSpec.groovy

```diff
@@ -0,0 +1,28 @@
+package test
+
+import geb.spock.GebReportingSpec
+import page.ExamPlannerHomePage
+import page.LoginPage
+import page.DashboardPage
+import groovy.json.JsonSlurper
+import spock.lang.Shared
+import helper.CommonHelper
+
+/**
+ * Created by jbdi on 2016-11-15.
+ */
+class LoginSpec extends GebReportingSpec {
+    @Shared jsonParser = CommonHelper.jsonToObject('src\\test\\resources\\values.json')
+
+    def "clicking 'Zaloguj się' button with correct credentials filled in forward to dashboard"() {
+        when:
```

Comment:
proponuje uzywac jezyk angileski w kodzie, jezeli to nie zwiazane z slownikami oraz tworzyc nazwy testow w formacie should ...
na przyklad: "should correct login and forward to dashboard page"